bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -62,14 +62,14 @@
            href="{{ plat.download_link }}"{% if plat.download_link_direct %}
            data-direct-link="{{ plat.download_link_direct }}"{% endif %}
            data-link-type="download"
-           data-display-name="{{ plat.os_arch_pretty or plat.os_pretty }}"
```

Comment:
Any specific reason why this is being removed?

Reply:
Oop, that was a mistake. good 👁️